app/uk/gov/hmrc/nationaldutyrepaymentcenter/models/AmendCaseResponseType.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.nationaldutyrepaymentcenter.models
+
+sealed trait AmendCaseResponseType
+
+  object AmendCaseResponseType extends Enumerable.Implicits {
+
+    case object Supportingdocuments extends WithName("supportingDocuments") with AmendCaseResponseType
+    case object Furtherinformation extends WithName("furtherInformation") with AmendCaseResponseType
```

Comment:
Can we use camel case here?

Reply:
It has been decided to do this in a future PR as a change will also be needed in the FE